src/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #dfdfdf;
+  min-height: 100vh;
+}
+
+.nav {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 15px 25px;
+  border: 2px solid black;
+  margin: 5px 8px;
+  border-radius: 5px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.nav a {
+  color: black;
+  padding: 0 10px;
+}
+
+.nav-border {
+  border-right: 2px solid black;
+  padding-right: 10px;
+}
+
+.nav a:hover {
+  color: blue;
+}
+
+main {
+  margin: 20px 0;
+  display: flex;
+  justify-content: center;
+}
+
+.bg {
+  padding: 1em;
+  border-bottom: #d2d2d2 solid 1px;
+}
+
+#todo-section {
+  background-color: #fcfafa;
+  width: 30em;
+  font-size: large;
+}
+
+#todo-section h2 {
+  color: #3d3d3d;
+  font-weight: 400;
+}
+
+#description,
+.borderless {
+  border: unset;
+  width: 20em;
+  padding: 0.2em;
+  font-size: large;
+  font-style: italic;
+  background-color: unset;
+}
+
+.todo label,
+.borderless,
+#clear label {
+  font-size: 1.2em;
+  padding-left: 1em;
+  color: #303030;
+}
+
+#description:active,
+#description:focus,
+.borderless:active,
+.borderless:focus {
+  outline: unset;
+}
+
+input[type='checkbox'] {
+  scale: 1.4;
+  cursor: pointer;
+}
+
+.todo {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
```

Comment:
- Currently, the checkbox is almost out of the border.  Kindly add the necessary  styling to it for a better look
- Please also consider adding more styling to the input box for a  better user experience. Kindly see the highlighted part below for clarity

![image](https://github.com/chudisoft/todo-list/assets/89091822/1fba5693-ac4d-46f6-9db4-75ffef0f5ef8)

Reply:
![image](https://github.com/chudisoft/todo-list/assets/119929069/d0296e2c-fece-479e-8e16-2334a91e31ed)
Am not sure why you got that error, but it looks as if the page is not properly loaded on your end.